.circleci/config.yml

```diff
@@ -13,7 +12,38 @@ jobs:
           command: yarn test
           name: Run tests via yarn
 
+jobs:
+  test:
+    executor:
+      name: node/default
+    steps:
+      - test_steps
+
+  test_build_and_publish:
+    executor:
+      name: node/default
+    steps:
+      - test_steps
+      - run:
+          command: yarn build
+          name: Compile TS to JS
+      - run:
+          name: Authenticate with registry
+          command: echo "//registry.npmjs.org/:_authToken=$npm_TOKEN" > ~/hub-react/.npmrc
+      - run:
+          name: Publish package
+          command: npm publish --dry-run
```

Comment:
Currently a `dry-run`, we'll need the publish token added into the Circle CI environment

Reply:
The reason for switching to `npm` from `yarn` for this step was that `yarn publish` didn't seem as good as npm's. It auto-increments the version in package.json before publishing, and is a bit more of a pain to auth via token (it tries to use login creds)

Reply:
👍